eris_test.go

```diff
@@ -260,40 +260,35 @@ func TestErrorFormatting(t *testing.T) {
 				t.Errorf("%v: expected { %v } got { %v }", desc, tc.output, err)
 			}
 
-			// todo: automate stack trace verification
 			_ = fmt.Sprintf("error formatting results (%v):\n", desc)
 			_ = fmt.Sprintf("%v\n", err)
 			_ = fmt.Sprintf("%+v", err)
 		})
 	}
 }
 
-func getFrames(frames []uintptr) []eris.StackFrame {
-	var sFrames []eris.StackFrame
-	for _, u := range frames {
-		pc := u - 1
-		fn := runtime.FuncForPC(pc)
-		if fn == nil {
-			frame := eris.StackFrame{
-				Name: "unknown",
-				File: "unknown",
-			}
-			sFrames = append(sFrames, frame)
-		}
-
-		name := fn.Name()
-		i := strings.LastIndex(name, "/")
-		name = name[i+1:]
-		file, line := fn.FileLine(pc)
+func getFrames(pc []uintptr) []eris.StackFrame {
+	var stackFrames []eris.StackFrame
+	if len(pc) == 0 {
+		return stackFrames
+	}
 
-		frame := eris.StackFrame{
+	frames := runtime.CallersFrames(pc)
```

Comment:
Is this function faster? what advanatage does it have over previous implementation?

Reply:
see #54 (and also https://golang.org/pkg/runtime/#Callers)